src/sentry/integrations/vsts/client.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import absolute_import
+
+from sentry.integrations.client import ApiClient
+
+UNSET = object()
+
+FIELD_MAP = {
+    'title': '/fields/System.Title',
+    'description': '/fields/System.Description',
+    'comment': '/fields/System.History',
+    'link': '/relations/-',
+}
+
+
+class VstsApiPath(object):
+    commits = 'https://{account_name}/DefaultCollection/_apis/git/repositories/{repo_id}/commits'
+    commits_batch = 'https://{account_name}/DefaultCollection/_apis/git/repositories/{repo_id}/commitsBatch'
+    commits_changes = 'https://{account_name}/DefaultCollection/_apis/git/repositories/{repo_id}/commits/{commit_id}/changes'
+    projects = 'https://{account_name}/DefaultCollection/_apis/projects'
+    repositories = 'https://{account_name}/DefaultCollection/{project}_apis/git/repositories/{repo_id}'
+    work_items = 'https://{account_name}/DefaultCollection/_apis/wit/workitems/{id}'
+    work_items_create = 'https://{account_name}/{project}/_apis/wit/workitems/${type}'
```

Comment:
should these all be unicode strings to avoid potential formatting errors?

Reply:
Yes plz.